src/metabase/api/search.clj

```diff
@@ -72,6 +72,8 @@
    ;; returned for Card and Dashboard
    :collection_position :integer
    :favorite            :boolean
+   ;; returned for everything except Collection
+   :updated_at          :timestamp
```

Comment:
we should add `created_at` and `updated_at` to Collection -- I think I forgot to add them in in the first place. It's part of #14877 anyway. (We don't have to do it in this PR tho)